augur/cli/util.py

```diff
@@ -64,11 +84,29 @@ def kill_processes():
     if processes != []:
         for process in processes:
             if process.pid != os.getpid():
-                logger.info(f"Killing process {process.pid}")
+                logger.info(f"Terminating process {process.pid}")
                 try:
                     process.send_signal(signal.SIGTERM)
+                    logger.info(f"sending SIGTERM Signal to {process.pid}")
```

Comment:
I could have saved a bunch of lines of code and just put the SIGKILL here by default, but I think there is merit to allowing some small amount of time for a softer termination first.

Reply:
This needs to be 2 separate commands, one for a graceful exit and one for a not-so-graceful exit. Having the kill command sleep each time when it's likely not even going to be needed in 99% of cases will just cause a lot of confusion. 15 seconds is also a really long time to be waiting in this case.